character-sfx.js

```diff
@@ -73,10 +104,28 @@ class CharacterSfx {
     this.preQ=new THREE.Quaternion();
     this.arr = [0, 0, 0, 0];
 
+    this.runStep = 0;
+    this.lastRunningTime = 0;
 
     
     this.oldNarutoRunSound = null;
   }
+  selectVoice(voicer) {

```

Comment:
This is duplicated code from the voice pack voicer.

It seems the voice pack voicer should be making this decision?